private_gpt/settings/settings.py

```diff
@@ -389,10 +412,12 @@ class Settings(BaseModel):
     ollama: OllamaSettings
     azopenai: AzureOpenAISettings
     vectorstore: VectorstoreSettings
+    graphstore: GraphStoreSettings | None = None
```

Comment:
use a non-nullable type here and add a `enabled` property instead, makes it easier to configure through env vars that way